Header header
HydrophoneSamples hydrophone_samples
float64 trigger_time # the time at which the ping was detected (relative to the center of the sample)
#   ie: ping data = 0.01 of a sampling that was 0.2 seconds in length
#   would imply that the ping happened 0.101 seconds from the start of the message
